szperacz/logger.py

```diff
@@ -1,18 +1,36 @@
 
 
 class Logger:
+    global log_level
+    log_level = 0
+    level = log_level
     DEBUG = '[DEBUG]'
     INFO = '[INFO]'
     ERROR = '[ERROR]'
 
+    error = open("error_log.txt", mode='w')
+
+    def set_log_level(self):
+        pass
+
+    def get_log_level(self):
+        return print(f'Logging level: {self.level}')
+
     def debug(self, msg):
-        return print(f'{self.DEBUG} {msg}')
+        if log_level == 0:
+            return print(f'{self.DEBUG} {msg}')
 
     def info(self, msg):
-        return print(f'{self.INFO} {msg}')
+        if log_level <= 1:
+            return print(f'{self.INFO} {msg}')
 
     def error(self, msg):
-        return print(f'{self.ERROR} {msg}')
+        if log_level <= 2:
+            error = open("error_log.txt", mode='a')
```

Comment:
Consider updating this using:

`with open(...)`

https://www.freecodecamp.org/news/with-open-in-python-with-statement-syntax-example/